### PR TITLE
Added count and last_page fields

### PR DESCRIPTION
**Pull request by e200**

### CODE REVIEW

File: example/Query/UsersQuery.php [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-use Rebing\GraphQL\Support\Query;
-use Rebing\GraphQL\Support\SelectFields;
 use GraphQL\GraphQL;
 use GraphQL\Type\Definition\Type;
-use Models\User; // not included in this project
-
-class UsersQuery extends Query {
+use Models\User;
+use Rebing\GraphQL\Support\Query;
+use Rebing\GraphQL\Support\SelectFields; // not included in this project
```
Comment (rebing) [inline example/Query/UsersQuery.php]: Comment in wrong place

File: example/Query/UserQuery.php [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+use GraphQL\GraphQL;
 use GraphQL\Type\Definition\Type;
+use Models\User;
 use Rebing\GraphQL\Support\Query;
-use GraphQL\GraphQL;
-use Rebing\GraphQL\Support\SelectFields;
-use Models\User; // not included in this project
-
-class UserQuery extends Query {
+use Rebing\GraphQL\Support\SelectFields; // not included in this project
```
Comment (rebing) [inline example/Query/UserQuery.php]: Comment in wrong place

File: example/Type/LocationType.php [hunk 3]
```diff
@@ -3,11 +3,11 @@
 namespace Rebing\GraphQL\Type\Location;
 
 use GraphQL\Type\Definition\Type;
-use Rebing\GraphQL\Support\Type as GraphQLType;
-use Models\Location; // not included in this project
-
-class LocationType extends GraphQLType {
+use Models\Location;
+use Rebing\GraphQL\Support\Type as GraphQLType; // not included in this project
```
Comment (rebing) [inline example/Type/LocationType.php]: Comment in wrong place

File: example/Type/UserType.php [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
-use Rebing\GraphQL\Support\Type as GraphQLType;
-use GraphQL\Type\Definition\Type;
 use GraphQL\GraphQL;
 use GraphQL\Privacy\MePrivacy;
-use Models\User; // not included in this project
-
-class UserType extends GraphQLType {
+use GraphQL\Type\Definition\Type;
+use Models\User;
+use Rebing\GraphQL\Support\Type as GraphQLType; // not included in this project
```
Comment (rebing) [inline example/Type/UserType.php]: Comment

File: phpunit.xml [hunk 5]
```diff
@@ -7,12 +7,11 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false"
-         syntaxCheck="false"
+         stopOnFailure="true"
```
Comment (rebing) [inline phpunit.xml]: Don't stop on failure

File: example/Query/UsersQuery.php (hunk 1 above)
Comment (e200) [reply example/Query/UsersQuery.php]: Fixed.

File: example/Query/UserQuery.php (hunk 2 above)
Comment (e200) [reply example/Query/UserQuery.php]: Fixed.

File: example/Type/LocationType.php (hunk 3 above)
Comment (e200) [reply example/Type/LocationType.php]: Fixed.

File: example/Type/UserType.php (hunk 4 above)
Comment (e200) [reply example/Type/UserType.php]: Fixed.

File: phpunit.xml (hunk 5 above)
Comment (e200) [reply phpunit.xml]: Fixed too.

**Comment by e200** [comment]
There's a lot of changes made on this pull request, please, leave it here for a while, I'll be back soon with some updates.

**Comment by e200** [comment]
Can you help me with these errors?

**Comment by e200** [comment]
Can you help me with these errors?

screenshot from 2019-02-14 15-53-11

**Comment by e200** [comment]
Sure, I'll close this pull request and chunk it.

About the style, it's a little bit hard to read the project code. Can we relay in a standard coding style like the [psr-2](https://www.php-fig.org/psr/psr-2) so we can easily contribute?

**Pull request closed without merging**